mta/tests/migration/controls/test_stakeholder_group.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_string
+
+
+@pytest.mark.sg
+def test_stakeholder_group_crud(stakeholder_group, get_api, delete_api, update_api):
+    # get stakeholder_group_from_db
+    stakeholder_group_from_db = get_api.stakeholders_id_get(stakeholder_group.id)
```

Comment:
.stakeholders_id_get is correct?
or 
.stakeholders_groups_id_get

Reply:
Thanks for noticing. Done